miniserver/src/data/datastore.js

```diff
@@ -0,0 +1,56 @@
+const Datastore = require('@google-cloud/datastore').Datastore
```

Comment:
use server's instead of duplicating

Reply:
Good point to remove duplication.
I've removed miniservers datastore code and am using servers.
@polymorpher if you approve I'll move this into a shared folder rather than miniserver linking to servers.

Reply:
Done in #14